atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -71,12 +71,13 @@ def __init__(self, tasker, workflow, parent_registry=None, parent_registry_insec
             metadata = self.workflow.builder.parent_images_digests
             metadata.update(parent_images_digests)
 
+        self.manifest_list_cache = {}
+
     def run(self):
         """
         Pull parent images and retag them uniquely for this build.
         """
         build_json = get_build_json()
-        self.manifest_list_cache = {}
```

Comment:
Removing such initialization from run, may cause issues when we use the same instance for multiple runs. However I don't think that this is atomic-reactor case. Hopefully somebody will correct me if I'm wrong.

Reply:
@twaugh @rcerven @athos-ribeiro

Reply:
yeah we always have just 1 plugin instance running

Reply:
Maybe the cache could just be cleared in `run()`, just to be sure?

Reply:
Yes, cache invalidation is one of the 4 most common programming issues. Please just to be clear it in run.